services/account-without-juggler/index.ts

```diff
@@ -1,39 +1,13 @@
-import { Application, Server } from 'loopback-next/packages/core';
+import { Application } from '@loopback/core';
 import { AccountController } from './controllers/AccountController';
 import { AccountRepository } from './repositories/account';
 
-class AccountMicroservice extends Application {
```

Comment:
I think it was intentional to have `AccountMicroservice extends Application`.  Is there any particular reason why you are preferring to use vanilla `Application` instance instead?